### PR TITLE
Adding batch per epoch option to the finetune script

### PR DESCRIPTION
**Pull request by daniel4x**
Allow to specify the batch per epoch in the finetune script (Probably missed it?)

### CODE REVIEW

**Comment by migalkin** [comment]
Thanks, that's a good catch - I was usually running fine-tuning with the `run_many` script to schedule several repeats with different seeds to get mean/std, but it can work in the single run as well

File: script/run.py [hunk 1]
```diff
@@ -289,12 +289,12 @@ def test(cfg, model, test_data, device, logger, filtered_data=None, return_metri
     val_filtered_data = val_filtered_data.to(device)
     test_filtered_data = test_filtered_data.to(device)
     
-    train_and_validate(cfg, model, train_data, valid_data, filtered_data=val_filtered_data, device=device, logger=logger)
+    train_and_validate(cfg, model, train_data, valid_data, filtered_data=val_filtered_data, device=device, ed_data, batch_per_epoch=cfg.train.batch_per_epoch, logger=logger)
```
Comment (migalkin) [inline script/run.py]: `ed_data` is apparently an artifact of your own code?

**Comment by daniel4x** [comment]
It was a copy-paste mistake 😄 .
I also missed the actual slicing from the for loop over the train data so the actual batch_per_epoch is still `len(train_loader)` as the variable is not used (it was only used for the print).

```Python
           # run.py training loop
            for batch in islice(train_loader, batch_per_epoch):
           ....
```

I'll create another patch for that, wdyt? @migalkin

**Comment by migalkin** [comment]
Sure, go ahead

**Pull request merged**